consult.el

```diff
@@ -2993,6 +2993,11 @@ INITIAL is the initial input."
 
 ;;;;; Command: consult-focus-lines
 
+(defsubst consult--extract-line-ind (matches)
+  "Extract and sort (line beg end) from MATCHES."
+  (sort (mapcar (apply-partially #'get-text-property 0 'line) matches)
+	(lambda (a b) (< (car a) (car b)))))
```

Comment:
Why do we have to sort? Aren't the lines already sorted?

Reply:
They are reversed via push, but that's an easy fix.  I was concerned that `completions-all-completions` doesn't have set the `display-sort-function` metadata when called, so that someone's completion style could interfere.  Is that a real issue?

Reply:
There is no issue with sorting. The filter function preserves the original order. So we should go with the reverse.

Reply:
Fixed.